spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe "/ingredients", type: :feature do
+  before(:each) do
+    @ing1 = Ingredient.create!(name: "Lemon", cost: 2)
+    @ing2 = Ingredient.create!(name: "Asparagus", cost: 1)
+    @ing3 = Ingredient.create!(name: "Salmon", cost: 10)
+  end
+
+  describe "ingredients index" do
+    it "displays all of the ingredients" do
+      visit "/ingredients"
+
+      expect(page).to have_content("#{@ing1.name}")
```

Comment:
Your view has `"Complexity: ..."` and "Genre" labels, your feature test should too.